mesa/batchrunner.py

```diff
@@ -51,8 +71,8 @@ def __init__(self, model_cls, parameter_values, iterations=1,
 
         """
         self.model_cls = model_cls
-        self.parameter_values = {param: self.make_iterable(vals)
-                                 for param, vals in parameter_values.items()}
+        self.variable_parameters = self._process_parameters(variable_parameters)
+        self.fixed_parameters = fixed_parameters or {}
```

Comment:
This is pylint w0102[1].

[1]http://pylint-messages.wikidot.com/messages:w0102